packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -142,7 +156,9 @@ contract Bridge is EssentialContract, IBridge {
             LibBridgeSend.isDestChainEnabled(AddressResolver(this), _chainId);
     }
 
-    function hashMessage(Message memory message) public pure returns (bytes32) {
+    function hashMessage(
+        Message calldata message
```

Comment:
Just for the sake of my understanding, is altering from memory to calldata to save on gas? I don't have a clear understanding of how it affects the function

Reply:
Yes